src/screens/home/views/ExposureDateView.tsx

```diff
@@ -0,0 +1,35 @@
+import React, {useMemo} from 'react';
+import {Text} from 'components';
+import {useI18n} from 'locale';
+import {useExposureNotificationService} from 'services/ExposureNotificationService';
+
+export const ExposureDateView = () => {
+  const i18n = useI18n();
+  const dateLocale = i18n.locale === 'fr' ? 'fr-CA' : 'en-CA';
+
+  const dateFormat =
```

Comment:
Order of the object doesn't matter here - it's determined by the `toLocaleString` function and depends on the locale. 
![Screen Shot 2020-11-10 at 12 12 31 PM](https://user-images.githubusercontent.com/5498428/98722171-45c6a680-234e-11eb-8d6d-ef9aa0a80267.png)